Mathlib/Algebra/Order/Group/Pointwise/Bounds.lean

```diff
@@ -48,6 +48,8 @@ lemma BddAbove.mul (hs : BddAbove s) (ht : BddAbove t) : BddAbove (s * t) :=
 lemma BddBelow.mul (hs : BddBelow s) (ht : BddBelow t) : BddBelow (s * t) :=
   (Nonempty.mul hs ht).mono (subset_lowerBounds_mul s t)
 
```

Comment:
Please split this out

Reply:
Okay sure... I've opened #19161, but the current PR is already tiny and will now for sure conflict with #19161